pom.xml

```diff
@@ -100,8 +100,8 @@
     <profile>
       <id>dev</id>
         <properties>
-          <omero.version>5.1.4-${ice.version}-SNAPSHOT</omero.version>
-          <bioformats.version>5.1.4-SNAPSHOT</bioformats.version>
+          <omero.version>5.2.0-${ice.version}-SNAPSHOT</omero.version>
```

Comment:
Oops. Sorry I let that slip but this `SNAPSHOT` version does not since all pre-5.2.0 releases had a suffix (i.e. `5.2.0-m1-${ice.version}-SNAPSHOT`). I will open a PR to bump this version to `5.2.1-${ice.version}-SNAPSHOT` once the first post 5.2.0 PR is merged.